_src/static/sass/utilities/_mixins.scss

```diff
@@ -0,0 +1,468 @@
+
+@mixin section-subtitle($type) {
+    @extend %heading;
+    @include font-size(18);
+    // font-family: $font-$type;
+}
+
+@mixin heading-general($type) {
+    @extend %heading;
+    @include font-size(16);
+    // font-family: $font-$type;
+}
+
+
+/* ============================
+6. Headings
+---------------------------- */
+%heading {
+    font-weight: 600;
+}
+
+
+/* ============================
+7. Flex Polyfill
+---------------------------- */
+%justify-right-flex {
+    -webkit-box-pack: flex-end;
+    -webkit-justify-content: flex-end;
+    -moz-box-pack: flex-end;
+    -ms-flex-pack: flex-end;
+    justify-content: flex-end;
+}
+
+%justify-left-flex {
+    -webkit-justify-content: flex-start;
+    -webkit-box-pack: flex-start;
+    -moz-box-pack: flex-start;
+    -ms-flex-pack: flex-start;
+    justify-content: flex-start;
+}
+
+%align-center-flex {
+    -webkit-align-items: center;
+    -webkit-flex-align: center;
+    -ms-flex-align: center;
+    align-items: center;
+}
+
+
+/* ============================
+8. Wrappers
+---------------------------- */
+%wrapper {
+    @include clearfix();
+    @include box-sizing(border-box);
+    width: 100%;
+}
+
+.col-wide  {
+    width: flex-grid(9, 12);
+    @include margin-right(flex-gutter());
+    @include float(left);
+}
+
+.col-narrow  {
+    width: flex-grid(3, 12);
+    @include float(left);
+}
+
+
+/* ============================
+9. Windows
+---------------------------- */
+%window {
+    @include clearfix();
+    border-radius: ($baseline / 10);
+    box-shadow: 0 1px 1px $shadow;
+    margin-bottom: $baseline;
+    border: 1px solid $gray-lighter;
+    background: $white;
+
+  // STATE: hover/active
+    &:hover, &:active {
+        box-shadow: 0 1px 1px $shadow;
+    }
+}
+
+
+/* ============================
+10. Visual Disabled
+---------------------------- */
+%disabled {
+    pointer-events: none;
+    outline: none;
+    cursor: default;
+}
+
+/* ============================
+11. Layering/Zindexing
+---------------------------- */
+%depth-0 { z-index: 1; }
+%depth-1 { z-index: 10; }
+%depth-2 { z-index: 100; }
+%depth-3 { z-index: 1000; }
+%depth-4 { z-index: 10000; }
+%depth-5 { z-index: 100000; }
+
+
+/* ============================
+12. Clear Children
+---------------------------- */
+%clear-first-child {
+
+    &:first-child {
+        margin-top: 0;
+        padding-top: 0;
+        border-top: none;
+    }
+}
+
+%clear-last-child {
+
+    &:last-child {
+        margin-bottom: 0;
+        padding-bottom: 0;
+        border-bottom: none;
+    }
+}
+
+/* ============================
+13. Buttons
+---------------------------- */
+@mixin button($type) {
+    @if $type == primary {
+        @extend %button-primary;
+    }
+
+    @if $type == secondary {
+        @extend %button-secondary;
+    }
+
+    @if $type == tertiary {
+        @extend %button-tertiary;
+    }
+}
+
+%button {
+    @extend %text-strong;
+    @include line-height();
+    @include box-sizing(border-box);
```

Comment:
Please remove, this is not needed as Bourbon has deprecated this utility (its natively supported enough now!) and our wildcard reset will take care of this.